### DISCUSSION TITLE
不好用，请求退款

### DISCUSSION THREAD

**Discussion by Alovera525** [getlantern/lantern-forum-cn#823] [General]
你好，不太好用，请求退款。
账号：QVera1225@foxmail.com
银联卡号支付

**Comment by MaxMKMax** [comment]
@Alovera525 您好. 请问您需要退款的具体原因是什么呢? 然后您需要退款的话还需要提供我支付单据. 
您要不再给我们一个机会? 希望您别退款, 可以继续用我们的服务. 如果那里我们做不好了, 我向您抱歉. 谢谢您支持蓝灯!

**Reply by Alovera525** [reply]
你好，当初买lantern的服务是因为，免费使用时发现它网速还行且可以用ChatGPT。
但是购买完服务以后，网速巨慢，instagram都刷不出来。。。感觉受骗了。
请问如果要退款的话，我是用信用卡预授权支付的，支付单指的是信用卡账单里的支付凭证吗？
如果不退款的话，你们有什么优惠政策或者可以多赠几个月服务么？

**Reply by Alovera525** [reply]
@Mike777Khoo 请求退款，请回答一下我的问题呀，谢谢！
我的诉求是VPN需要：1. Stable, 2. 流畅, 3. 电脑/手机能用，且ChatGPT之类的外网都能上
但是Lantern服务购买以后，就经常不流畅也不稳定，今天甚至都连不上，而且每次一连VPN，微信什么的就用不了。
所以请求退款，请问我银行卡预授权支付的，是否截图信用卡里的支付凭证即可？Thanks

**Reply by MaxMKMax** [reply]
@Alovera525 嗯嗯. 您提供截图就可以了. 只要有那个购买的单号就行了. 
我会帮您申请 2 个月免费给您吧只要您不退款.

**Reply by Alovera525** [reply]
@Mike777Khoo 我在银行App里查询了一下没有购买单号，但是打银行电话要到了单号，你看一下是否可以，谢谢！
交易编号：1725 3306 1390 0225 6635 2003
收单行流水号：2019 0004 2024 0903 1020 5373 6902

**Reply by Alovera525** [reply]
@Mike777Khoo 你好，我这边还是要求退款的，请你查看一下我提供的交易编号和收单行流水号，谢谢！

**Reply by Alovera525** [reply]
> @Alovera525 嗯嗯. 您提供截图就可以了. 只要有那个购买的单号就行了. 我会帮您申请 2 个月免费给您吧只要您不退款.

@Mike777Khoo, Mike，请尽快回复，你要的单号我上周就已经回复你了，谢谢！

**Reply by Alovera525** [reply]
@reflog @anacrolix @myleshorton @Mike777Khoo 
Anyone please deal with my refund request ASAP? I first request this three weeks ago, it last long.
I have provided all the information requested as above. I need a refund cause I'm NOT satisfied with the service Lantern provide.

**Reply by MaxMKMax** [reply]
@Alovera525 Sorry for the unsatisfaction and inconvenience caused. We will improve our product again soon. Hope it will meet your expectations in the future.

I already help you to submit to the person in-charge to proceed refund. 

Feel free to come back to Lantern when you found we had make improve. Thank you for supporting Lantern. 

_____

很抱歉给您带来的不满意和不便。我们将很快再次改进我们的产品。希望它在未来能满足您的期望。

我已经帮您提交给负责人进行退款。

如果您发现我们有改进，请随时回到 Lantern。感谢您对 Lantern 的支持。

**Reply by Alovera525** [reply]
Thanks, @Mike777Khoo, how long the refund process will be finished by estimation plz?

**To improve the product, I would only consider using Lantern in the future if the following issues are addressed:
1. Improve and stabilize network speed on iPhone.
2. Resolve connection difficulties on iPhone.
3. Ensure Lantern does not interfere with other apps, such as WeChat (currently, I am unable to receive WeChat messages while using Lantern).
4. Improve On-call Service SLA **

cc @reflog @anacrolix @myleshorton

**Reply by MaxMKMax** [reply]
@Alovera525 Thanks for the feedback. Usually will take 1 week - 2 weeks to complete the refund. I will help you to follow up again, no worries. We will help you to settle the issues. 

Since you have some issues in using the app, can you help us to do an in app report? So we can have the log and make improvement. Thanks. 

您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by Alovera525** [reply]
@Mike777Khoo, report submitted as {77368064}, thanks!

**Reply by greySkiesMarsh** [reply]
@Alovera525 Your refund has been processed, and should be reflected in your account witin 5-10 business days.
